README.md

```diff
@@ -73,6 +77,9 @@ Options:
 * `body`, it can be a `String`, a `Buffer`, `Uint8Array` or a `stream.Readable`.
 * `headers`, an object with header-value pairs.
 * `signal`, either an `AbortController` or an `EventEmitter`.
+* `requestTimeout`, the timeout after which a request will time out, in
+  milliseconds.  Use `0` to disable it entirely.
```

Comment:
I think `null` for disabling is more intuitive and normal.

Reply:
I see your point, but using `0` to disable a numeric configuration is a well-established standard in the community IMHO.

Reply:
Yep, looking through the node docs you are right.